tests/integ/scala/test_dataframe_aggregate_suite.py

```diff
@@ -910,6 +910,38 @@ def test_multiple_column_distinct_count(session):
     res.sort(key=lambda x: x[0])
     assert res == [Row("a", 2), Row("x", 1)]
 
+    aa = [
+        "a1",
+        "a1",
+        "a1",
+        "a1",
+        "a2",
+        "a2",
+        "a2",
+        "a3",
+        "a3",
+        "a4",
+    ]
+
+    bb = [
+        "b1",
+        "b2",
+        "b2",
+        "b3",
+        "b1",
+        "b2",
+        "b5",
+        "b1",
+        "b4",
+        "b1",
+    ]
+
+    df = session.create_dataframe([[a, b] for a, b in zip(aa, bb)], ["a", "b"])
```

Comment:
nit: Unneeded comprehension.
```suggestion
    df = session.create_dataframe(list(zip(aa, bb)), ["a", "b"])
```